doc/api/inspector.md

```diff
@@ -0,0 +1,72 @@
+# Inspector
+
+> Stability: 1 - Experimental
+
+The `inspector` module provides an API for interacting with the V8 inspector.
+
+It can be accessed using:
+
+```js
+const inspector = require('inspector');
+```
+
+## Class: inspector.Session
+<!-- YAML
+added: REPLACEME
+-->
+
+The `inspector.Session` class is a primary means of dispatching the messages to
+the V8 inspector backend and receiving message responses and notifications.
+
+An exception will be thrown if a session is created when there is already
+an ongoing inspector session (either created through this API or by connecting
+to a WebSocket port).
+
+### Event 'protocolNotification'
+<!-- YAML
+added: REPLACEME
+-->
+
+* {Object} The notification
+
+Emitted when a notification from an inspector backend was received.
+
+It is also possible to subscribe to a specific inspector notification type by
+specifying the method. E.g. this snippet will print the reason for application
+been suspended when the [Debugger.paused][debugger-paused] event is recieved
+be called whenever the program execution is suspended:
+```js
+session.on('Debugger.paused', ({reason}) => console.log(reason));
+```
+
+### session.post(method[, params][, callback])
+<!-- YAML
+added: REPLACEME
+-->
+
+* method {string}
+* params {Object}
+* callback {Function}
+
+Posts a message to the inspector backend. Optional `callback` will be notified
+when a response is received. `callback` is a function that accepts two
+optional arguments - error and message-specific result.
+
+```js
+session.post('Runtime.evaluate', {'expression': '2 + 2'},
+             (error, {result}) => console.log(result.value));
+```
+
+Up-to-date version of the V8 inspector protocol is published on the
+[protocol viewer][protocol-viewer]
+
+### session.disconnect()
+<!-- YAML
+added: REPLACEME
+-->
+
+Immediately closes the session. Some pending callbacks may not receive
+the responses they are waiting for.
+
+[debugger-paused]: https://chromedevtools.github.io/debugger-protocol-viewer/v8/Debugger/#event-paused
```

Comment:
Link is out of date, it should be https://chromedevtools.github.io/devtools-protocol/v8/Debugger/#event-paused